src/pretix/plugins/banktransfer/views.py

```diff
@@ -274,6 +291,33 @@ def get_context_data(self, **kwargs):
         return ctx
 
 
+class BankTransactionFilterForm(forms.Form):
+    search_text = forms.CharField(required=False, widget=forms.TextInput(attrs={'class': "form-control", "placeholder": _("Search text")}))
+    amount_min = forms.DecimalField(required=False, widget=forms.NumberInput(attrs={'class': "form-control", "placeholder": _("min"), "size": 8}))
+    amount_max = forms.DecimalField(required=False, widget=forms.NumberInput(attrs={'class': "form-control", "placeholder": _("max"), "size": 8}))
```

Comment:
This will work badly with locales like German where you want to enter numbers with comma. Fix: pass ``localize=True`` to the field and don't use NumberInput but a normal TextInput instead